src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
 }
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-inline: 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  color: #000;
+  text-decoration: none;
+  height: 60px;
+
+  padding: 0;
+  margin: 0;
+
+  text-transform: uppercase;
+  font: inherit;
+}
+
+.nav__link:hover,
+.is-active {
+  color: var(--main-color);
+}
+
+.nav__item {
+  margin-right: 20 px;
```

Comment:
We don't need it and it is broken, please, remove
![image](https://github.com/mate-academy/layout_moyo-header/assets/46566190/cd4dd08a-d5e7-4d71-998d-4c0eb4176e00)